### ISSUE TITLE
sourmash lca database link in tutorial is wrong

### ISSUE THREAD

**Issue by taylorreiter** [dib-lab/sourmash#632]
Sourmash tutorial link:
https://sourmash.readthedocs.io/en/latest/tutorials-lca.html

This link is broken:
```
curl -L -o genbank-k31.lca.json.gz https://osf.io/zskb9/download
```

The proper link is:
```
curl -L -o genbank-k31.lca.json.gz https://osf.io/4f8n3/download
```

Currently here:
https://github.com/dib-lab/sourmash/blob/master/doc/tutorials-lca.md

**Comment by taylorreiter** [comment]
See  #633

**Comment by ctb** [comment]
This was fixed in #633 and will be further updated in #631.
